sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/LocalRelation.scala

```diff
@@ -79,7 +79,8 @@ case class LocalRelation(
   }
 
   override def computeStats(): Statistics =
-    Statistics(sizeInBytes = EstimationUtils.getSizePerRow(output) * data.length)
+    Statistics(sizeInBytes = EstimationUtils.getSizePerRow(output) * data.length,
+      rowCount = Some(data.size))
```

Comment:
Post-hoc comment: not a big deal but I just noticed that we have `data.length` and `data.size` used on adjacent lines and they mean the same thing. It would have been better if we use the same method for the same thing, at least on the same page of code.

Reply:
Good point, thank you, will file a followup to fix.